Still a minor include error, should be able to wrap this up tomorrow:

```
[+3922 ms] C:\Users\Jonah\Documents\flutter-uwp-template\src\winuwp\runner_uwp\flutter_runner.cpp(30,10):
fatal error C1083: Cannot open include file: 'flutter/flutter_windows.h': No such file or directory
[C:\Users\Jonah\Documents\flutter-uwp-template\src\build\winuwp\runner_uwp\helloreloaded.vcxproj]
[  +92 ms] C:\Program Files (x86)\Windows
```

--- a/packages/flutter_tools/lib/src/build_system/targets/windows.dart
+++ b/packages/flutter_tools/lib/src/build_system/targets/windows.dart
@@ -167,6 +167,13 @@ class UnpackWindowsUwp extends Target {
         platform: TargetPlatform.windows_x64
       )
     );
+    // Copy flutter_windows.h into flutter directory as well.
+    final File flutterWindows = outputDirectory.childFile('flutter_windows.h');
+    final File flutterWindowsDest = flutterWindows.parent.parent.childFile('flutter_windows.h');
+    flutterWindows.copySync(flutterWindowsDest.path);
+    depfile.outputs.add(flutterWindowsDest);
+    //
+
     final DepfileService depfileService = DepfileService(
       fileSystem: environment.fileSystem,
       logger: environment.logger,

--- a/packages/flutter_tools/lib/src/plugins.dart
+++ b/packages/flutter_tools/lib/src/plugins.dart
@@ -1192,7 +1192,7 @@ Future<void> injectPlugins(
   bool linuxPlatform = false,
   bool macOSPlatform = false,
   bool windowsPlatform = false,
-  bool winuwpPlatform = false,
+  bool winUwpPlatform = false,
   bool webPlatform = false,
 }) async {
   final List<Plugin> plugins = await findPlugins(project);
@@ -1213,7 +1213,7 @@ Future<void> injectPlugins(
   if (windowsPlatform) {
     await _writeWindowsPluginFiles(project, plugins);
   }
-  if (winuwpPlatform) {
+  if (winUwpPlatform) {
     await _writeWindowsUwpPluginFiles(project, plugins);
   }
   if (!project.isModule) {

--- a/packages/flutter_tools/lib/src/project.dart
+++ b/packages/flutter_tools/lib/src/project.dart
@@ -279,7 +279,7 @@ class FlutterProject {
       macOSPlatform: featureFlags.isMacOSEnabled && macos.existsSync(),
       windowsPlatform: featureFlags.isWindowsEnabled && windows.existsSync(),
       webPlatform: featureFlags.isWebEnabled && web.existsSync(),
-      winuwpPlatform: featureFlags.isWindowsUwpEnabled && windowsUwp.existsSync(),
+      winUwpPlatform: featureFlags.isWindowsUwpEnabled && windowsUwp.existsSync(),
     );
   }
 
@@ -292,7 +292,7 @@ class FlutterProject {
     bool macOSPlatform = false,
     bool windowsPlatform = false,
     bool webPlatform = false,
-    bool winuwpPlatform = false,
+    bool winUwpPlatform = false,
   }) async {
     if (!directory.existsSync() || hasExampleApp || isPlugin) {
       return;
@@ -316,7 +316,7 @@ class FlutterProject {
     if (webPlatform) {
       await web.ensureReadyForPlatformSpecificTooling();
     }
-    if (winuwpPlatform) {
+    if (winUwpPlatform) {
       await windowsUwp.ensureReadyForPlatformSpecificTooling();
     }
     await injectPlugins(
@@ -327,7 +327,7 @@ class FlutterProject {
       macOSPlatform: macOSPlatform,
       windowsPlatform: windowsPlatform,
       webPlatform: webPlatform,
-      winuwpPlatform: winuwpPlatform,
+      winUwpPlatform: winUwpPlatform,
     );
   }
 

--- a/packages/flutter_tools/test/commands.shard/hermetic/build_windows_test.dart
+++ b/packages/flutter_tools/test/commands.shard/hermetic/build_windows_test.dart
@@ -107,7 +107,7 @@ void main() {
     bool verbose = false,
     void Function() onRun,
     String stdout = '',
-    bool winuwp = true,
+    bool winuwp = false,
   }) {
     return FakeCommand(
       command: <String>[

--- a/packages/flutter_tools/test/general.shard/build_system/targets/windows_test.dart
+++ b/packages/flutter_tools/test/general.shard/build_system/targets/windows_test.dart
@@ -182,6 +182,7 @@ void main() {
     expect(fileSystem.file(r'C:\winuwp\flutter\ephemeral\flutter_plugin_registrar.h'), exists);
     expect(fileSystem.file(r'C:\winuwp\flutter\ephemeral\flutter_texture_registrar.h'), exists);
     expect(fileSystem.file(r'C:\winuwp\flutter\ephemeral\flutter_windows.h'), exists);
+    expect(fileSystem.file(r'C:\winuwp\flutter\flutter_windows.h'), exists);
     expect(fileSystem.file('C:\\winuwp\\flutter\\ephemeral\\$icuData'), exists);
     expect(fileSystem.file('C:\\winuwp\\flutter\\ephemeral\\$windowsCppClientWrapper\\foo'), exists);
 
@@ -220,6 +221,7 @@ void main() {
       r'C:\winuwp\flutter\ephemeral\flutter_plugin_registrar.h',
       r'C:\winuwp\flutter\ephemeral\flutter_texture_registrar.h',
       r'C:\winuwp\flutter\ephemeral\flutter_windows.h',
+      r'C:\winuwp\flutter\flutter_windows.h',
       'C:\\winuwp\\flutter\\ephemeral\\$icuData',
       'C:\\winuwp\\flutter\\ephemeral\\$windowsCppClientWrapper\\foo',
     ]));

